tracing-mock/src/subscriber.rs

```diff
@@ -31,15 +194,41 @@ pub fn mock() -> MockSubscriberBuilder {
     }
 }
 
+/// Create a [`MockSubscriberBuilder`] with a name already set.
+///
+/// This constructor is equivalent to `subscriber::mock().named(name)`.
```

Comment:
maybe worth linking to the `named` method here?

Reply:
Done. I also included the example from the `named` method modified to use this constructor.